### DISCUSSION TITLE
Additional support to the community!

### DISCUSSION THREAD

**Discussion by mrrajeshrai** [medplum/medplum#4857] [General]
Nice work with this project.

We've created simple repositories with same mission to accelerate AI medical application development using AI and related information. Would love to collaborate. Here is the link: https://github.com/GlobalHealthAI/ModelLibrary.

Please review the work and let us know you feedback!

Thanks

**Comment by codyebberson** [comment]
Hi @mrrajeshrai - thanks for sharing!  Congrats, and best of luck on your project!
